apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -53,6 +60,30 @@ export const CreateOrder = () => {
     orderTypeChanged(index === 0 ? 'market' : 'limit')
   }
 
+  const {createUnsignedTx, isLoading} = useSwapTx({
+    onSuccess: (yoroiUnsignedTx) => {
+      unsignedTxChanged(yoroiUnsignedTx)
+      swap()
+      setShowLimitPriceWarning(false)
+    },
+    onError: (error) => {
+      console.log(error)
```

Comment:
please check with product about the error here, if the tx submission fails, we navigate to the error summary, but I don't see a state if the tx creation fails.